schema/person-statement.json

```diff
@@ -105,16 +105,30 @@
     },
     "birthDate": {
       "title": "Date of birth",
-      "description": "The date of birth for this individual. Please provide as precise a date as possible in ISO 8601 format. When only the year or year and month is known, these can be given as YYYY or YYYY-MM.",
+      "description": "The date of birth for this individual, in YYYY, YYYY-MM, or YYYY-MM-DD format.",
       "type": "string",
-      "pattern": "^([\\+-]?\\d{4}(?!\\d{2}\b))((-?)((0[1-9]|1[0-2])(\\3([12]\\d|0[1-9]|3[01]))?|W([0-4]\\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\\d|[12]\\d{2}|3([0-5]\\d|6[1-6])))([T\\s]((([01]\\d|2[0-3])((:?)[0-5]\\d)?|24\\:?00)([\\.,]\\d+(?!:))?)?(\\17[0-5]\\d([\\.,]\\d+)?)?([zZ]|([\\+-])([01]\\d|2[0-3]):?([0-5]\\d)?)?)?)?$",
+      "anyOf": [
+        {
+          "pattern": "^(\\d{4})(-(1[0-2]|0[1-9]))?$"
```

Comment:
@rhiaro - is this definitely right? Testing it in a couple of online validators it won't match simple dates. Will "^([0-9]{4})(-(1[0-2]|0[1-9]))?$" do the job?

Reply:
`\d` is equivalent to `[0-9]`. The extra `\` in front is needed as an escape in the json schema pattern matching, but won't work if you paste it into online validators. This regex matches only yyyy and yyyy-mm, as the rest is covered by the `date` format in the `anyOf`. I've triple checked, this definitely works as expected.

Reply:
Ah - it didn't occur to me that the escape `\` wasn't a regex escape but a json schema/python escape. I get it.